src/components/Navbar/__test__/TopNavbar.test.tsx

```diff
@@ -1,9 +1,38 @@
 import React from "react";
 import { render, screen } from "@testing-library/react";
 import TopNavbar from "../TopNavbar";
+import { BrowserRouter } from "react-router-dom";
 
-it("should render nav links on top navigation bar", () => {
-  render(<TopNavbar />);
-  const linkElement = screen.getByText(/New Cars/i);
-  expect(linkElement).toBeInTheDocument();
+const MockNavbar = () => {
+  return (
+    <BrowserRouter>
+      <TopNavbar />
+    </BrowserRouter>
+  );
+};
+describe("Navbar", () => {
+  it("should render all cars text in the Navbar", () => {
+    render(<MockNavbar />);
+    const textElement = screen.getByText(/ALL CARS/i);
+    expect(textElement).toBeInTheDocument();
+  });
+
+  it("should render profile in the Navbar", () => {
+    render(<MockNavbar />);
+    const ProfileElement = screen.getByText(/Profile/i);
+    expect(ProfileElement).toBeInTheDocument();
+  });
+
+  it("should render logotext in the Navbar", () => {
+    render(<MockNavbar />);
+    const LogoElement = screen.getByText(/XTREME CARS/i);
+    expect(LogoElement).toBeInTheDocument();
+  });
+
```

Comment:
we also need to include tests that, check that the links are taking the users to the right page